content/blog/2021-04-15-april-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,190 @@
+﻿---
+title: April '21 Heartbeat
+date: 2021-04-16
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires, and more!
+descriptionLong: |
+  Monthly updates are here! Read all about new lessons and a tutorials from
+  our community, DVC's use in longevity research, our CEO's interviews and 
+  talk at the Toronto Machine Learning Society, new team members, and more!
+picture: 2021-04-16/april21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-heartbeat/718
+tags:
+  - Heartbeat
+  - DVC
+  - MLOps
+  - Made with ML
+  - AI Singapore
+  - DAGsHub
+  - Toronto Machine Learning Society
+---
+
+## From the Community
+
+We're starting with the community this month because it has been overflowing
+with great content from our users. It's like we're on a sugar high!
+
+![Sugar High](https://media.giphy.com/media/oiGCnybFPh6Q8/giphy.gif)
+
+### Goku Mohandas' New Lessons!
+
+First up, [Goku Mahandas](https://twitter.com/GokuMohandas) of
+[Made With ML](https://madewithml.com/) has added this
+[Versioning Lesson](https://madewithml.com/courses/mlops/versioning/) to the
+popular **MLOps Course** using DVC.  
+It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
+
+<external-link
+href="https://madewithml.com/courses/mlops/"
+title="MLOps - Versioning Code, Data and Models"
+description="Using DVC to version data and models for reproducibility 
+in a local storage use case"
+link="https://madewithml.com/"
+image="/uploads/images/2021-04-15/made-with-ml-logo.png"/>
+
+### Ryzal Kamis Tutorial
+
+[**Ryzal Kamis**](https://www.linkedin.com/in/ryzalkamis/) of
+[AI Singapore](https://twitter.com/AISingapore) has created an
+[**in depth tutorial**](https://makerspace.aisingapore.org/2021/04/data-versioning-for-cd4ml-part-2/)
+on data versioning using DVC. This is a follow up article to his
+[tutorial that was featured in the September Heartbeat.](https://dvc.org/blog/september-20-dvc-heartbeat)
+Thanks Ryzal for this detailed work! 🙏🏼
+
+<external-link
+href="https://makerspace.aisingapore.org/2021/04/data-versioning-for-cd4ml-part-2/"
+title="Data Versioning for CD4ML - Part 2"
+description="DVC tutorial"
```

Comment:
we still have the same `DVC tutorial` description in all these snippets (except the first one). Could we please make them a bit longer and specific to the item?